imports/plugins/core/shipping/client/templates/checkout/shipping.html

```diff
@@ -16,21 +16,32 @@ <h3 class="panel-title">
               <span class="pull-right">{{formatPrice rate}}</span>
             </li>
           {{else}}
-            <div class="panel-body">
-              {{#if isAdmin}}
-                <div class="alert alert-danger">
-                  <span data-i18n="checkoutShipping.noShippingMethods">No shipping methods are configured.</span>
-                  <a href="#" data-event-action="configure-shipping">
-                    <span data-i18n="checkoutShipping.configureNow">Configure now.</span>
-                  </a>
+            {{#if hasQueriedShipmentMethods}}
+              <div class="panel-body">
+                {{#if isAdmin}}
+                  <div class="alert alert-danger">
+                    <span data-i18n="checkoutShipping.noShippingMethods">No shipping methods are configured.</span>
+                    <a href="#" data-event-action="configure-shipping">
+                      <span data-i18n="checkoutShipping.configureNow">Configure now.</span>
+                    </a>
+                  </div>
+                {{else}}
+                  <div class="alert alert-danger">
+                    <span data-i18n="checkoutShipping.noShippingMethods">No shipping methods are available for this shop.</span>
+                    <span data-i18n="checkoutShipping.contactAdmin">Please contact the store administrator.</span>
+                  </div>
+                {{/if}}
+              </div>
+            {{else}}
+              <div>
+                <h4>
+                  Loading shipping methods... Please wait...
```

Comment:
This needs an i18n entry. 

@rymorgan some design direction here would be good.  I think I'd prefer to only see a loader.

I cringe to see this much Blaze updated without a React conversion.